src/database/append_smiley.rs

```diff
@@ -0,0 +1,61 @@
+use crate::database::models::{Restaurant, SmileyReport};
+use serde::Serialize;
+
+#[derive(Debug, Clone, PartialEq, Serialize)]
+pub struct RestaurantWithSmileyReport {
+    pub id: i32,
+    pub smiley_restaurant_id: i32,
+    pub name: String,
+    pub address: String,
+    pub zipcode: String,
+    pub city: String,
+    pub cvr: String,
+    pub pnr: String,
+    pub latitude: f32,
+    pub longitude: f32,
+    pub version_number: i32,
+    pub smileyreports: Vec<SmileyReport>,
+}
+
+pub fn convert_res_smiley_pairs(
+    mut input: Vec<(Restaurant, SmileyReport)>,
+) -> Vec<RestaurantWithSmileyReport> {
```

Comment:
Hvorfor er der indeksering på 0 og 1 osv? Måske kunne funktionen konverteres eller simplificeres til mindre funktioner.

Reply:
querien laver en tuple af restaurant og smileyreport. hvilket gør at index 0 er restaurant elementet i den tuple og 1 er smiley reporten for den restaurant. 

Vi skal bruge det hele i funktionen til at kunne indsætte alle smileyreporterne som en vektor for hver restaurant. så tror ikke at det kan opsplittes på en god måde.